### DISCUSSION TITLE
Campaign and Shuttles - We should be able to buy and play shuttles in Campaign!

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#5136] [Ideas and suggestions]
We should be able to play and buy shuttles in Campaign. It'd be cool if we can have it for either campaign respawn (with limited use) or shuttles that we can send it out for EVAs or attach to any submarines. We just need to also be able to buy transducers and navigation terminal for those if they are drones.

**Comment by Udrakan** [comment]
Being able to start campaign on a shuttle would be nice, perhaps with a warning as it was in the past.

**Comment by ArcAngelica** [comment]
I second this, I had designed a submarine that could both be used as a shuttle as much as it could be used as a campaign submarine, but currently if set to be a Shuttle it can't be used nor bought in campaign, and if set to a Submarine Class it can't be selected as a shuttle.

This problem would be fixed if you could just select shuttles in campaign, allowing me to properly set it to be a Shuttle.

**Comment by mLuby** [comment]
> We just need to also be able to buy transducers and navigation terminal for those if they are drones.

I'd recommend this be a separate proposal, and for now just keep drones out of it by adding a "Drone" designation, meaning a shuttle that's not intended for crew.

Also shuttles should listed _below_ the normal submarines in their own list item group with a "Shuttles" heading, so players are less likely to pick one thinking it's a full sub.
